### PR TITLE
zh-TW: Translate android/build-rules.md

### PR DESCRIPTION
**Pull request by enshuoh**
Part of https://github.com/google/comprehensive-rust/issues/684.

### CODE REVIEW

File: po/zh-TW.po [hunk 1]
```diff
@@ -9781,11 +9781,11 @@ msgstr ""
 
 #: src/android/build-rules.md:1
 msgid "# Build Rules"
-msgstr ""
+msgstr "# 建構規則"
 
 #: src/android/build-rules.md:3
 msgid "The Android build system (Soong) supports Rust via a number of modules:"
-msgstr ""
+msgstr "Android 建構系統 (Soong) 透過以下模組支援 Rust："
```
Comment (mingyc) [inline po/zh-TW.po:9788]: How about adding `的`: `Android 的建構系統`

File: po/zh-TW.po [hunk 2]
```diff
@@ -9811,10 +9811,30 @@ msgid ""
...
+"|\n"
+"| `rust_library`    | 生成一個 Rust 函式庫，及其對應的 `rlib` 和 `dylib` "
+"變體。                                     |\n"
+"| `rust_ffi`        | 生成一個可被 `cc` 模組使用的  Rust C 函式庫，及其對應的"
+"靜態和共享變體。                           |\n"
+"| `rust_proc_macro` | 生成一個 `proc-macro` Rust 函式庫，類似於編譯器 "
+"擴充。                                      |\n"
+"| `rust_test`       | 使用Rust自動化測試框架，生成一個 Rust "
+"測試檔。                                   |\n"
+"| `rust_fuzz`       | 利用 `libfuzzer` 生成一個Rust模糊測試的二進位 "
```
Comment (mingyc) [inline po/zh-TW.po:9828]: Add space in between Rust: `生成一個 Rust 的模糊測試的二進位執行檔`

File: po/zh-TW.po [hunk 3]
```diff
@@ -9811,10 +9811,30 @@ msgid ""
...
+"執行檔。                                   |\n"
+"| `rust_protobuf`   | 對指定的protobuf，生成其定義界面的 Rust "
+"代碼及函式庫 。                            |\n"
```
Comment (mingyc) [inline po/zh-TW.po:9831]: > Generates source and produces a Rust library that provides an interface for a particular protobuf. 
> 對指定的protobuf，生成其定義界面的 Rust 代碼及函式庫 。           

`生成對應 protobuf 介面的 Rust 原始碼及函式庫 。`

File: po/zh-TW.po [hunk 4]
```diff
@@ -9811,10 +9811,30 @@ msgid ""
...
+"| `rust_bindgen`    | 生成用於連接 C 函式庫的 Rust "
+"代碼及函式庫。                             |"
```
Comment (mingyc) [inline po/zh-TW.po:9833]: `代碼` -> `原始碼`

**Review by mingyc — CHANGES_REQUESTED** [review]

File: po/zh-TW.po (hunk 4 above)
Comment (mgeisler) [reply po/zh-TW.po:9833]: I forget if I already showed you the suggestion feature (left-most button in the toolbar above):

```suggestion
"原始碼及函式庫。                             |"
```

That allows the PR author (or yourself!) to incorporate the change with a single click.

**Review by victorhsieh — CHANGES_REQUESTED** [review]

**Comment by mgeisler** [comment]
Hi @victorhsieh, feel free to use the suggestion feature here to fix up the things you feel confident about. You can then apply the fixes directly and merge the PR. That often saves a lot of round-trips for the translations.

File: po/zh-TW.po [hunk 5]
```diff
@@ -9811,10 +9811,30 @@ msgid ""
 "| `rust_bindgen`    | Generates source and produces a Rust library "
 "containing Rust bindings to C libraries.              |"
 msgstr ""
+"| 模組型態          | "
+"敘述 "
+"|\n"
+"|-------------------|----------------------------------------------------------------------------------------------------|\n"
+"| `rust_binary`     | 生成一個 Rust 二進位執行檔。"
```
Comment (victorhsieh) [inline po/zh-TW.po:9818]:
```suggestion
"| `rust_binary`     | 生成一個 Rust 執行檔。"
```

File: po/zh-TW.po [hunk 6]
```diff
@@ -9811,10 +9811,30 @@ msgid ""
...
+"|\n"
+"| `rust_library`    | 生成一個 Rust 函式庫，及其對應的 `rlib` 和 `dylib` "
+"變體。                                     |\n"
+"| `rust_ffi`        | 生成一個可被 `cc` 模組使用的  Rust C 函式庫，及其對應的"
```
Comment (victorhsieh) [inline po/zh-TW.po:9822]:
```suggestion
"| `rust_ffi`        | 生成一個可被 `cc` 模組使用的 Rust C 函式庫，及其對應的"
```

File: po/zh-TW.po [hunk 7]
```diff
@@ -9811,10 +9811,30 @@ msgid ""
...
+"靜態和共享變體。                           |\n"
+"| `rust_proc_macro` | 生成一個 `proc-macro` Rust 函式庫，類似於編譯器 "
+"擴充。                                      |\n"
+"| `rust_test`       | 使用Rust自動化測試框架，生成一個 Rust "
+"測試檔。                                   |\n"
+"| `rust_fuzz`       | 利用 `libfuzzer` 生成一個 Rust 模糊測試的二進位 "
```
Comment (victorhsieh) [inline po/zh-TW.po:9828]:
```suggestion
"| `rust_fuzz`       | 生成一個使用 libfuzzer 的 Rust 模糊測試"
```

Comment (hueich) [reply po/zh-TW.po:9828]: Remember to add the backticks.

**Review by hueich — COMMENTED** [review]
Agree with Victor's suggestions.

File: po/zh-TW.po (hunk 7 above)
Comment (mgeisler) [reply po/zh-TW.po:9828]: There, I added the backticks :smile: 

```suggestion
"| `rust_fuzz`       | 生成一個使用 `libfuzzer` 的 Rust 模糊測試"
```

**Comment by mgeisler** [comment]
Hey all! Let's get this merged so you can start seeing it in context of the rest of the book. That would make it easier to apply any small corrections needed (and it would make it easier for you to [refresh the translation](https://github.com/google/comprehensive-rust/blob/main/TRANSLATIONS.md#refreshing-an-existing-translation)).

**Comment by mgeisler** [comment]
@victorhsieh and @mingyc, you asked for changes in your review — please take another quick look.

**Review by victorhsieh — APPROVED** [review]
I had one open, nit-picking comment, but am ok to merge without it.

**Comment by mgeisler** [comment]
> I had one open, nit-picking comment, but am ok to merge without it.

Thanks! I just converted your comment straight into an issue: #1080. Then it should be possible to revisit after merging this and after upgrading mdbook-i18n-helpers.

File: po/zh-TW.po [hunk 8]
```diff
@@ -9811,10 +9811,30 @@ msgid ""
...
+"|\n"
+"|-------------------|----------------------------------------------------------------------------------------------------|\n"
+"| `rust_binary`     | 生成一個 Rust 二進位執行檔。"
+"|\n"
+"| `rust_library`    | 生成一個 Rust 函式庫，及其對應的 `rlib` 和 `dylib` "
+"變體。                                     |\n"
+"| `rust_ffi`        | 生成一個可被 `cc` 模組使用的  Rust C 函式庫，及其對應的"
+"靜態和共享變體。                           |\n"
+"| `rust_proc_macro` | 生成一個 `proc-macro` Rust 函式庫，類似於編譯器 "
+"擴充。                                      |\n"
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/zh-TW.po:9825]: Moved to #1080.

**Comment by mgeisler** [comment]
> Sorry for dropping the ball. Lost my attention to the thread waiting for review.

All good! :smile: A lot of things have been in flight here so it's hard to stay on top of it all.

> It seems Victor hasn't gotten the CLA, blocking the merge.

Yeah, though Victor is a Googler so the problem is the use of a private email address. I'll override it and then we should be able to merge.

**Pull request merged**